src/engine/SCons/Action.xml

```diff
@@ -35,70 +35,85 @@ executed to build targets.
 By default, SCons will add
 to each target
 an implicit dependency on the command
-represented by the first argument on any
-command line it executes.
+represented by the first argument of any
+command line it executes (which is typically
+the command itself). By setting such
+a dependency, &SCons; can determine that
+a target should be rebuilt if the command changes,
+such as when a compiler is upgraded to a new version.
 The specific file for the dependency is
 found by searching the
 <varname>PATH</varname>
 variable in the
-<varname>ENV</varname>
-environment used to execute the command.
+<varname>ENV</varname> dictionary
+in the &consenv; used to execute the command.
+The default is the same as
+setting the &consvar;
+&cv-IMPLICIT_COMMAND_DEPENDENCIES;
+to a true value (<literal>True</literal>,
+any non-zero number, etc.).
```

Comment:
or 1, not any non-zero, 2 or above would yield different functionality right?

Reply:
Eh, you're quite right. That sort of violates the usual Python-ism that 0 is False, anything else is True since it uses >1 for something else. Have to word that more carefully.